data/src/Cardano/Data/StrictMaybe.hs

```diff
@@ -0,0 +1,100 @@
+{-# LANGUAGE DeriveAnyClass #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE DeriveTraversable #-}
+
+{- | Strict version of the 'Maybe' type..
+-}
+module Cardano.Data.StrictMaybe
+  ( StrictMaybe (SNothing,SJust),
+    -- * Conversion: StrictMaybe <--> Maybe
+    strictMaybeToMaybe,
+    maybeToStrictMaybe,
+    -- * accessing the optional component
+    fromSMaybe,
+  )
+  where
+
+import Data.Aeson (FromJSON (..), ToJSON (..))
+import NoThunks.Class (NoThunks (..))
+import GHC.Generics (Generic)
+import Control.DeepSeq (NFData)
+import Cardano.Binary
+  ( FromCBOR (fromCBOR),
+    ToCBOR (toCBOR),
+    decodeBreakOr,
+    decodeListLenOrIndef,
+    encodeListLen,
+  )
+import qualified Control.Monad.Fail
+import Data.Default.Class(Default(..))
```

Comment:
For consistency with the rest of the file:
```suggestion
import Data.Default.Class (Default (..))
```

Reply:
done